programs/vote/src/vote_state/mod.rs

```diff
@@ -166,33 +166,24 @@ pub fn to<T: WritableAccount>(versioned: &VoteStateVersions, account: &mut T) ->
 fn set_vote_account_state(
     vote_account: &mut BorrowedAccount,
     vote_state: VoteState,
-    feature_set: &FeatureSet,
 ) -> Result<(), InstructionError> {
-    // Only if vote_state_add_vote_latency feature is enabled should the new version of vote state be stored
-    if feature_set.is_active(&feature_set::vote_state_add_vote_latency::id()) {
-        // If the account is not large enough to store the vote state, then attempt a realloc to make it large enough.
-        // The realloc can only proceed if the vote account has balance sufficient for rent exemption at the new size.
-        if (vote_account.get_data().len() < VoteStateVersions::vote_state_size_of(true))
-            && (!vote_account
-                .is_rent_exempt_at_data_length(VoteStateVersions::vote_state_size_of(true))
-                || vote_account
-                    .set_data_length(VoteStateVersions::vote_state_size_of(true))
-                    .is_err())
-        {
-            // Account cannot be resized to the size of a vote state as it will not be rent exempt, or failed to be
-            // resized for other reasons.  So store the V1_14_11 version.
-            return vote_account.set_state(&VoteStateVersions::V1_14_11(Box::new(
-                VoteState1_14_11::from(vote_state),
-            )));
-        }
-        // Vote account is large enough to store the newest version of vote state
-        vote_account.set_state(&VoteStateVersions::new_current(vote_state))
-    // Else when the vote_state_add_vote_latency feature is not enabled, then the V1_14_11 version is stored
-    } else {
-        vote_account.set_state(&VoteStateVersions::V1_14_11(Box::new(
+    // If the account is not large enough to store the vote state, then attempt a realloc to make it large enough.
+    // The realloc can only proceed if the vote account has balance sufficient for rent exemption at the new size.
+    if (vote_account.get_data().len() < VoteStateVersions::vote_state_size_of(true))
```

Comment:
I'm thinking for a future change we introduce a feature flag to stop accepting the old vote state version. This will allow us to finally remove the boolean from `vote_state_size_of`.

Reply:
Yeah that sounds great